sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -1730,9 +1730,8 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
 
   test("SPARK-9083: sort with non-deterministic expressions") {
     import org.apache.spark.util.random.XORShiftRandom
```

Comment:
Can we move this import into the top? (this is not related to this pr though)

Reply:
@maropu will do.